treeherder/seta/analyze_failures.py

```diff
@@ -24,6 +25,7 @@ def __init__(self, **options):
 
     def run(self):
         fixed_by_commit_jobs = get_failures_fixed_by_commit()
+        load_preseed()
```

Comment:
Since the SETA load_preseed management command is called from the treeherder load_initial_data management command (at the first ever treeherder startup), how about we modify load_preseed so that it calls analyze_failures first before preseeding?  In load_preseed we could add:

call_command('analyze_faiures')

Immediately before calling treeherder.seta.preseeed.load_preseed.

Thoughts?